java/java.platform/src/org/netbeans/modules/java/platform/queries/PlatformSourceForBinaryQuery.java

```diff
@@ -94,24 +94,46 @@ public SourceForBinaryQueryImplementation2.Result findSourceRoots2(URL binaryRoo
             this.cache.put (binaryRoot, res);
             return res;
         }
-        String binaryRootS = binaryRoot.toExternalForm();
-        if (binaryRootS.startsWith(JAR_FILE)) {
-            if (binaryRootS.endsWith(RTJAR_PATH)) {
-                //Unregistered platform
-                String srcZipS = binaryRootS.substring(4,binaryRootS.length() - RTJAR_PATH.length()) + SRC_ZIP;
-                try {
-                    URL srcZip = FileUtil.getArchiveRoot(new URL(srcZipS));
-                    FileObject fo = URLMapper.findFileObject(srcZip);
-                    if (fo != null) {
-                        return new UnregisteredPlatformResult (fo);
-                    }
-                } catch (MalformedURLException mue) {
-                    Exceptions.printStackTrace(mue);
-                }
-            }
+        return searchUnregisteredPlatform(binaryRoot.toExternalForm());
+    }
+
+  static SourceForBinaryQueryImplementation2.Result searchUnregisteredPlatform(String binaryRootS) {
+    String srcZipS = null;
+    String srcZipIn = null;
+    if (binaryRootS.startsWith(JAR_FILE)) {
+      if (binaryRootS.endsWith(RTJAR_PATH)) {
```

Comment:
would be good to use standard indentation settings for new code.

Reply:
It would help if NetBeans apisupport projects specified the proper indentation settings. Otherwise the source gets formatted by my _global defaults_ (which are set to Google Java standards, not Sun/Oracle now).

Reformatted in [5b0d432](https://github.com/apache/netbeans/pull/5157/commits/5b0d432b4d1e0b87ad92486b56e90cfe80c8d1f2)

Reply:
awesome thanks.

Yeah its probably a good idea to copy a few selected IDE defaults to the project settings. Its probably not worth it though if it has to be set up for every single module, would be too much noise for little gain IMO.

Reply:
A project type can provide some default `AuxiliarySettings` - [java.mx.project module](https://github.com/apache/netbeans/blob/master/java/java.mx.project/src/org/netbeans/modules/java/mx/project/SuiteProperties.java#L54) is doing that to influence the Eclipse formatter (used for mx projects).

Reply:
NetBeans Formatter needs one more level of indirection, as it currently uses the project's auxiliary properties directly. If I had time...